lib/core/src/lib/form/components/widgets/amount/amount.widget.html

```diff
@@ -10,24 +10,23 @@
         >
     </div>
     <div>
-        <mat-form-field class="adf-amount-widget__input adf-form-field-input" [hideRequiredMarker]="true" [floatLabel]="placeholder ? 'always' : 'auto'">
-            <mat-label class="adf-label" *ngIf="!field.leftLabels" [attr.for]="field.id"
-                >{{field.name | translate }}<span class="adf-asterisk" [style.visibility]="isRequired() ? 'visible' : 'hidden'">*</span></mat-label
-            >
+        <mat-form-field class="adf-amount-widget__input adf-form-field-input" [floatLabel]="placeholder ? 'always' : 'auto'">
+            @if (field.name || this.field?.required) { <mat-label class="adf-label" *ngIf="!field.leftLabels" [attr.for]="field.id">{{field.name | translate }}</mat-label> }
             <span matTextPrefix class="adf-amount-widget__prefix-spacing">{{ currency }}&nbsp;</span>
             <input
                 matInput
                 [title]="field.tooltip"
                 class="adf-input"
                 type="text"
                 [id]="field.id"
-                [required]="isRequired()"
+                [required]="field.required && field.isVisible"
                 [placeholder]="placeholder"
                 [value]="field.value"
                 [(ngModel)]="field.value"
                 (ngModelChange)="onFieldChanged(field)"
                 [disabled]="field.readOnly"
                 (blur)="markAsTouched()"
+                [required]="isRequired()"
```

Comment:
you set required 2 times, and last one rewrites the previous, making the whole change as before